backend/utils/base.py

```diff
@@ -177,30 +174,17 @@ def get_search_kwargs(self) -> dict:
                 return {"k": self.k, "fetch_k": self.fetch_k, "filter": self.filter}
 
 
-class SearchQuery(BaseModel):
+class DefaultQueryInput(BaseModel):
     collection_name: str = Field(
         default=None,
         title="Collection name on which to search",
     )
-    retrieval_chain_name: Literal["RetrievalQA", "CustomRetrievalQA"] = Field(
-        default="RetrievalQA",
-        title="Name of the retrieval chain to use for retrieving documents",
-    )
+
     retriever_config: RetrieverConfig = Field(
         title="Retriever configuration",
     )
     query: str = Field(title="Question to search for", max_length=1000)
     model_configuration: LLMConfig
-    system_prompt: str = Field(
```

Comment:
Interesting, why is this not needed anymore?

Reply:
this was used for custom retriever mostly